test/ctia/test_helpers/core.clj

```diff
@@ -249,8 +249,18 @@
                        "ctia.http.port" http-port
                        "ctia.http.show.port" http-port]
        (let [config (build-transformed-init-config)
+             services-map (cond-> (init/default-services-map config)
+                            (#{:threatgrid} (get-in config [:ctia :auth :type]))
+                            ;; dynamic requires can be removed when #'with-properties is phased out or moved
```

Comment:
Here we take advantage of services being a map from keywords to services to stub what we want.